packages/gas-fee-controller/package.json

```diff
@@ -68,7 +69,8 @@
     "typescript": "~4.8.4"
   },
   "peerDependencies": {
-    "@metamask/network-controller": "^18.0.0"
+    "@metamask/network-controller": "^18.0.0",
+    "@metamask/preferences-controller": "^9.0.0"
```

Comment:
Is this dependency necessary? I'm not seeing it used below. Specifically adding a peer dependency would be a breaking change since clients would now have to use this version for compatibility. But if we don't need to have this then it doesn't need to be a breaking change.

Reply:
oh, yeah, that was included by mistake

Reply:
fixed in ffc64cca